shared/app/global-errors/index.desktop.tsx

```diff
@@ -181,7 +98,7 @@ const styles = Kb.Styles.styleSheetCreate(() => {
     overflow: 'hidden',
     position: 'absolute',
     right: 0,
-    top: 0,
+    top: 40,
```

Comment:
there is a dead area on top thats unclickable due to electron and we need to move down for this to actually work